src/event_loop.rs

```diff
@@ -520,52 +514,28 @@ unsafe impl rwh_05::HasRawDisplayHandle for OwnedDisplayHandle {
 }
 
 /// Used to send custom events to [`EventLoop`].
-pub struct EventLoopProxy<T: 'static> {
-    event_loop_proxy: platform_impl::EventLoopProxy<T>,
-}
-
-impl<T: 'static> Clone for EventLoopProxy<T> {
-    fn clone(&self) -> Self {
-        Self { event_loop_proxy: self.event_loop_proxy.clone() }
-    }
+#[derive(Clone)]
+pub struct EventLoopProxy {
+    event_loop_proxy: platform_impl::EventLoopProxy,
 }
 
-impl<T: 'static> EventLoopProxy<T> {
-    /// Send an event to the [`EventLoop`] from which this proxy was created. This emits a
-    /// `UserEvent(event)` event in the event loop, where `event` is the value passed to this
-    /// function.
+impl EventLoopProxy {
+    /// Wake up the [`EventLoop`] resulting in [`user_wake_up`] being called.
     ///
-    /// Returns an `Err` if the associated [`EventLoop`] no longer exists.
+    /// When multiple wake ups are sent they'll result only in a single [`user_wake_up`] call.
     ///
```

Comment:
```suggestion
    ///
    /// If the event loop has is no longer running, this is effectively a no-op.
    ///
```

Reply:
I'll put it for now, but I'm thinking of returning the error for proxy, because the queue on the main thread stuff could be done via proxy as well, and it'll also need error handling, etc. I think.

For now should be good without error to move further with it.